service-front/module/Application/view/application/pages/cp/how_will_the_cp_confirm.twig

```diff
@@ -137,23 +137,22 @@
                 <input class="govuk-radios__input" id="PostOffice" name="id_method" type="radio" value="POST_OFFICE">
                 <label class="govuk-label govuk-radios__label" for="PostOffice">
                     Post Office
-                    <p class="govuk-hint govuk-!-margin-top-0">
-                        UK passport (up to 18 months expired), current UK photocard driving licence
-                        or current international photo ID.
-                    </p>
 
                     <details class="govuk-details govuk-!-margin-top-4 govuk-!-margin-bottom-0">
                         <summary class="govuk-details__summary">
                                 <span class="govuk-details__summary-text">
-                                    Which international documents are accepted
+                                    Documents accepted at the Post Office
                                 </span>
                         </summary>
                         <div class="govuk-details__text">
                             <ul class="govuk-list govuk-list--bullet govuk-hint">
+                                <li>UK passport (up to 18 months expired)</li>
+                                <li>UK photocard driving licence</li>
+                                <li>UK biometric residence permit</li>
+                                <li style="list-style-type:' '"></li>
```

Comment:
just having an empty <li> kept the bullet-point so have had to change the list-style-type. If i set it to `none` then it didn't leave a gap so i've come up with this - feels a bit janky so there may well be a cleaner way i'm not aware of?

Reply:
Yeah not adding HTMl to fix visual issues is a good rule to follow in genera. The semantics matter and empty `li`s will confuse assistive tech. You might need to pad either the` li` or `ul` with a spacing class. https://design-system.service.gov.uk/styles/spacing/ has some depending on what you are trying to achieve.